pom.xml

```diff
@@ -35,6 +35,8 @@
     <packaging>pom</packaging>
 
     <properties>
+        <asciidoctor.plugin.version>2.2.3</asciidoctor.plugin.version>
```

Comment:
This is a major upgrade, which upgrades to an Asciidoctor version which is more ahead than the downstream version of Asciidoctor, therefore please keep the old version here. 

I'm aligning the versions a bit with https://github.com/keycloak/keycloak/pull/20266, but not so much. Again, please revert the changes on `pom.xml` for this PR. It needs a bit more discussion.